#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
bool estop
bool ignition_manual
bool ignition_auto
bool drive_status
uint8 ev_mode
uint8 state_of_charge
uint16 battery_power
bool rf_estop